.travis.yml

```diff
@@ -16,7 +16,11 @@ env:
     - ROS_DISTRO=melodic
   matrix:
     - TEST=clang-format,catkin_lint
-    - DOCKER_IMAGE=moveit/moveit:master-source
+      UPSTREAM_WORKSPACE=.rosinstall/random_pick/random_pick.rosinstall
+
+    - DOCKER_IMAGE=xiaotaoqi/moveit_example_apps:random_pick
```

Comment:
Do we need this separate docker or can this inherit from the moveit ones with some additional support?

Reply:
Sometimes CI times out if directly builds from the moveit ones, so I use this docker image to reduce the CI build time. It is built from the docker file `.docker/random_pick/Dockerfile`, which also builds on top of the moveit-source docker image. I hope new application could also provide their docker file. If the docker files of this repo can build automatically, I'm thinking this docker image could be replaced by the `moveit/moveit_example_apps:random_pick`.